docs/lab_project_list.md

```diff
@@ -0,0 +1,23 @@
+## lab project list
```

Comment:
The docs are actually Auto generated during release to keep them aligned to the latest released version. Can you remove this file?